mitosheet/src/mito/components/toolbar/FormulaTabContents.tsx

```diff
@@ -101,13 +108,19 @@ export const FormulaTabContents = (
     }
 
     return (<div className='mito-toolbar-bottom'>
+        <ToolbarButton action={props.actions.buildTimeActions[ActionEnum.Set_Column_Formula]} />
+        <ToolbarButton action={props.actions.buildTimeActions[ActionEnum.Add_Column]} />
+
+        <div className="toolbar-vertical-line"/>
+
         {(Object.keys(formulaCategories) as FunctionCategory[]).map((category: FunctionCategory) => {
             // We don't want to display the finance category in the toolbar because we don't currently have any finance functions
             if (category === 'FINANCE') {
                 return null;
             }
             return getFormulaDropdown(formulaCategories[category], category)
         })}
+        {getFormulaDropdown(ActionEnum.Formulas_Dropdown_Custom, 'custom')}
```

Comment:
I like that we display if there no formulas -- it's great for enterprise discoverability!